extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/WorkerImpl.java

```diff
@@ -1319,7 +1342,7 @@ private void writeDurableStorageSuccessFileIfNeeded(final int stageNumber)
             e,
             "Unable to create the success file [%s] at the location [%s]",
             DurableStorageUtils.SUCCESS_MARKER_FILENAME,
-            DurableStorageUtils.getSuccessFilePath(
+            DurableStorageUtils.getWorkerOutputSuccessFilePath(
```

Comment:
Should we get the worker output path from the DurableStorageOutputChannelFactory instead?